Adding debugging documentation (#166)
ONSdigital/dp-compose: v2/stacks/auth/README.md

- copy the highlighted values

Note that the AWS_SESSION_TOKEN is only valid for 12 hours. Once the token has expired you would need to stop the stack, retrieve and set new credentials before running the stack again.

> # Example on how to debug a Go application running in docker 

Reviewer: I'd be tempted to put this in the parent directory's `README.md` because it's not specific to `auth`

Author: This is now addressed.

This example will show how we can debug `dp-identity-api` when we run the auth stack.

> - Copy and paste the following code into the `Dockerfile.local` file in `dp-identity-api` repo

Reviewer: Sorry, but it wasn't clear if you were adding to - or replacing - the file contents.

```suggestion
- Replace the contents of the `Dockerfile.local` file (in the `dp-identity-api` repo) with the following code
```

Author: This is now addressed.
